benchmark/src/main/java/ai/timefold/solver/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -316,10 +316,11 @@ public List<String> getWarningList() {
                     + " Maybe reduce the parallelBenchmarkCount.");
         }
         EnvironmentMode environmentMode = plannerBenchmarkResult.getEnvironmentMode();
-        if (environmentMode != null && environmentMode.isAsserted()) {
+        if (environmentMode != null && environmentMode.isStepAssertOrMore()) {
+            // Phase assert performance impact is negligible.
             warningList.add("The environmentMode (" + environmentMode + ") is asserting."
                     + " This decreases performance."
-                    + " Maybe set the environmentMode to " + EnvironmentMode.REPRODUCIBLE + ".");
+                    + " Maybe set the environmentMode to " + EnvironmentMode.PHASE_ASSERT + ".");
```

Comment:
Maybe `NO_ASSERT`?

Reply:
No. `PHASE_ASSERT` is the new default. But the message needs to be reworded.